src/workspace/workspace-wrapper.component.css

```diff
@@ -9,15 +9,37 @@
   display: block;
   animation: fade_in_show 0.3s;
   width: 50vw;
+  height: 80%;
+  overflow-y: scroll;
 }
 
 @keyframes fade_in_show {
   0% {
     opacity: 0;
     transform: scale(0);
   }
+
   100% {
     opacity: 1;
     transform: scale(1);
   }
 }
+
+#optionsBar {
+  width: 100% !important;
+  background-color: #eff1f2;
+  position: fixed;
+  z-index: 2;
```

Comment:
z-index are always a red flag, and becomes extremely difficult to debug as the app grows and you have competing `z-index`. Why are you using the `z-index` here?

Reply:
The input fields scrolled above the title. But it's okay, i'll find a way around it without the z-index. Could you please also review at the next commit?